modules/fedimint-wallet-tests/tests/tests.rs

```diff
@@ -585,3 +585,349 @@ fn build_wallet_server_configs(
     )?;
     Ok((wallet_cfg.into_values().collect(), client_cfg))
 }
+
+#[cfg(test)]
```

Comment:
Is this `cfg` attribute still strictly needed?

Reply:
yes, we shouldn't need cfg(test) in -tests crate